// ------------------------------------------------------------
// Copyright 2022 Youyuan Wu
// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
// ------------------------------------------------------------

syntax = "proto3";

package fabricrpc;

message request_header {
  string url = 1;
}

message reply_header {
  int32 status_code = 1;
  string status_message = 2;
}